articles/quickstart/spa/_includes/_install_auth0js.md

```diff
@@ -1,8 +1,11 @@
 ## Integrate Auth0 in your Application
 
-### Install auth0.js
+### Loading auth0.js
```

Comment:
Could we just remove this heading "Loading auth0.js" since there are no other steps to "Integrate Auth0 in your Application"?

Integrate
- Load via dependencies
- Load via CDN

Reply:
Never mind, I see there is more in the main document